src/black/comments.py

```diff
@@ -157,9 +160,18 @@ def make_comment(content: str) -> str:
     if (
         content
         and content[0] == "\N{NO-BREAK SPACE}"
-        and not content.lstrip().startswith("type:")
+        and not is_type_comment_string("# " + content.lstrip(), mode=mode)
     ):
         content = " " + content[1:]  # Replace NBSP by a simple space
+    if (
+        Preview.standardize_type_comments in mode
+        and content
+        and "\N{NO-BREAK SPACE}" not in content
+        and is_type_comment_string("# " + content.lstrip(), mode=mode)
+    ):
+        type_part, value_part = content.strip().split(":", 1)
```

Comment:
```suggestion
        type_part, value_part = content.split(":", 1)
```
The `.strip()` here is unnecessary as both parts will be stripped individually